lib/libc/gen/psignal.c

```diff
@@ -61,3 +63,70 @@ psiginfo(const siginfo_t *si, const char *s)
 {
 	psignal(si->si_signo, s);
 }
+
+int
+sig2str(int signum, char *str)
+{
+	int n;
+
+	if (signum <= 0 || signum > SIGRTMAX)
+		return (-1);
+
+	if (signum < sys_nsig)
+		(void) strcpy(str, sys_signame[signum]);
```

Comment:
No space between ')' and 's'.  There and everywhere you do the cast to ignore result.